CodeStarterCamp_Week2/randomLottoNumber.swift

```diff
@@ -8,16 +8,37 @@
 import Foundation
 
 // 로또 당첨 번호를 생성하는 함수
-func makeLottoNumbers() -> Set<Int> {
+func makeLottoNumbers() {
     var lottoNumbers: Set<Int> = Set<Int>()
     var lottoNumber: Int
     while lottoNumbers.count < 6 {
         lottoNumber = Int.random(in: 1...45)
         lottoNumbers.insert(lottoNumber)
     }
-    return lottoNumbers
+    // 생성한 로또 당첨 번호를 저장
+    saveLottoNumbers(newLottoNumbers: lottoNumbers)
 }
 
+// 로또 당첨 번호를 저장하는 함수
+func saveLottoNumbers(newLottoNumbers: Set<Int>) {
```

Comment:
동일한 의미의 이름은 사용을 지양하는 것이 좋습니다!
이렇게 바꿔도 어떤 기능을 하는 함수인지 바로 파악할 수 있을 것 같아요. 🤔
```suggestion
func save(lottoNumbers: Set<Int>) {
```